fdb-relational-api/fdb-relational-api.gradle

```diff
@@ -53,11 +53,15 @@ buildHost=${InetAddress.getLocalHost().getCanonicalHostName()}
 }
 
 dependencies {
+    api project(':fdb-extensions')
+    annotationProcessor project(':fdb-extensions')
```

Comment:
There's a bit of a problem here, in that `fdb-extensions` has the FDB client as a transitive dependency, which we don't want to take on in the `fdb-relational-api` project (as it's used by the client subproject). One possible solution would be to split out the stability annotations out of `fdb-extensions` into their own standalone sub-project (`fdb-api-annotations` or something) that all of the other projects can then add as a dependency. Or we hold off on adding API stability annotations in `fdb-relational-api`, `fdb-relational-grpc`, and `fdb-relational-jdbc`, which I think are the three subprojects that we don't want to have a dependency on `fdb-java`.